cypress/component/Forms/forms.spec.jsx

```diff
@@ -410,17 +410,14 @@ describe('FormField - Tests', () => {
     });
 
     it('should remove single on click', () => {
-      cy.spy(props, 'onChange');
+      cy.spy(props, 'onChange').as('onChange');
       mount(<FormField {...props}/>);
       cy.get('#dataCustodianEmail').type('a@a.com');
       cy.get('#dataCustodianEmail').type('{enter}');
-      const firstPill = cy.get('.formField-dataCustodianEmail .pill').first();
```

Comment:
Linter didn't like this assignment.  Refactored to remove it.

Reply:
Yes, this is a common pattern we use in tests, thank you